socorro/external/boto/crashstorage.py

```diff
@@ -98,8 +96,6 @@ def __init__(
         self.bucket = bucket
         self.dump_file_suffix = dump_file_suffix
 
-        self.metrics = markus.get_metrics(metrics_prefix)
```

Comment:
This never got used in the crashstorage classes except for `ESCrashStorage`. I removed it from everywhere and for `ESCrashStorage` I did something different.